coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -1093,8 +1094,11 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
 
   /**
    * Local helper to cancel a director that was edited.
+   * @param id Id of the director being edited.
    */
-  private cancelEditDirector (): void {
+  private cancelEditDirector (id = null): void {
+    if (id) this.restoreDirName(id - 1)
```

Comment:
Should check that id > 0 otherwise "id - 1" could be negative. You could also move this check to restoreDirName().

PS null == 0

Reply:
Great idea, i will update this.

Reply:
Updated 👍